### PR TITLE
Migrate mode for InfluxDB

### PR DESCRIPTION
**Pull request by hagen1778**
```
 go build && ./vmctl influx --help                                                                   :(
NAME:
   vmctl influx - Migrate timeseries from InfluxDB

USAGE:
   vmctl influx [command options] [arguments...]

OPTIONS:
   --influx-addr value              Influx server addr (default: "http://localhost:8086")
   --influx-user value              Influx user [$INFLUX_USERNAME]
   --influx-password value          Influx user password [$INFLUX_PASSWORD]
   --influx-database value          Influx database
   --influx-retention-policy value  Influx retention policy (default: "autogen")
   --influx-series-filter value     Influx filter expression to select timeseries. E.g. "FROM cpu WHERE arch='x64'"
   --influx-chunk-size value        The chunkSize defines max amount of series to be returned in one chunk (default: 10000)
   --vm-addr value                  VictoriaMetrics address to perform import requests. Should be the same as --httpListenAddr value for single-node version or VMSelect component. (default: "http://localhost:8428")
   --vm-account-id value            Account(tenant) ID - for the cluster VM only (default: -1)
   --vm-concurrency value           Number of workers concurrently performing import requests to VM (default: 2)
   --vm-compress                    Whether to apply gzip compression to import requests (default: true)
   --help, -h                       show help (default: false)


 go build && ./vmctl influx --influx-database benchmark                                              
InfluxDB import mode
2020/01/16 22:06:54 Exploring scheme for database "benchmark"
2020/01/16 22:06:54 fetching fields: command: "show field keys"; database: "benchmark"; retention: "autogen"
2020/01/16 22:06:54 found 10 fields
2020/01/16 22:06:54 fetching series: command: "show series"; database: "benchmark"; retention: "autogen"
Found 1000 timeseries to import. Continue? [Y/n] y
30 / 1000 [---->___________________________________________________________________] 3.00% 8 p/s
```

### CODE REVIEW

**Comment by valyala** [comment]
Feel free merging the pull request when it is ready

**Comment by valyala** [comment]
Oops, clicked incorrect button :)

File: influx/influx.go [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+
+	result := make([]string, len(values["fieldKey"]))
+	for i, v := range values["fieldKey"] {
+		result[i] = v.(string)
+	}
+	log.Printf("found %d fields", len(result))
+	return result, nil
+}
+
+func (c *Client) getSeries(filter string) ([]string, error) {
```
Comment (tenmozes) [inline influx/influx.go]: unused parameter

File: main.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
+package main
```
Comment (tenmozes) [inline main.go]: is it better to move it to cmd/vmctl?

Comment (hagen1778) [reply main.go]: it was decided to keep it as is

**Pull request merged**